packages/router-bridge/package.json

```diff
@@ -4,11 +4,12 @@
   "description": "Bridge on top of cmf-router or react-router v5 + connected-react-router",
   "main": "lib/index.js",
   "scripts": {
-    "build:dev": "talend-scripts build:lib:umd --dev",
-    "build:prod": "talend-scripts build:lib:umd --prod",
+    "build:dev": "talend-scripts build --umd --dev",
+    "build:prod": "talend-scripts build --umd --prod",
     "pre-release": "yarn build:dev && yarn build:prod",
-    "build:lib": "talend-scripts build:lib",
-    "build": "talend-scripts build:lib",
+    "build:lib": "talend-scripts build",
```

Comment:
Looks like we can keep only one command build because build:lib and build are doing the same thing

Reply:
yes and build is never used by the CI